app/components/admin/schools/cohort_component.html.erb

```diff
@@ -59,6 +59,7 @@
           summary_list.with_row do |row|
             row.with_key(text: "Appropriate body")
             row.with_value(text: school_cohort&.appropriate_body&.name || "No appropriate body")
+            row.with_action(text: "Change", visually_hidden_text: "appropriate body", href: change_appropriate_body_href)
```

Comment:
This is my fault for not thinking about this before, but as there can be multiple of these links per page, they need to be disambiguated for screen reader users (using links mode) - so we should include the year within the visually-hidden text.

```suggestion
            row.with_action(text: "Change", visually_hidden_text: "appropriate body for their #{ cohort.start_year} programme", href: change_appropriate_body_href)
```

Reply:
Updated